app/components/services/servicesController.js

```diff
@@ -72,15 +72,28 @@ function ($scope, $stateParams, $state, Service, ServiceHelper, Messages, Pagina
 
   function fetchServices() {
     $('#loadServicesSpinner').show();
-    Service.query({}, function (d) {
-      $scope.services = d.map(function (service) {
-        return new ServiceViewModel(service);
+    $q.all({
+      services: Service.query({}).$promise,
+      tasks: Task.query({filters: {'desired-state': ['running']}}).$promise,
+      nodes: Node.query({filters: {membership: ['accepted']}}).$promise,
```

Comment:
I don't think this is possible, I just tried it and the docker engine returns `{"message":"Invalid filter 'Availability'"}`. I based my implementation on the information here: https://docs.docker.com/engine/api/v1.26/#tag/Node.